hedera-node/hedera-network-admin-service-impl/src/test/java/com/hedera/node/app/service/networkadmin/impl/test/handlers/NetworkGetExecutionTimeHandlerTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.networkadmin.impl.test.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.hedera.hapi.node.base.QueryHeader;
+import com.hedera.hapi.node.base.ResponseHeader;
+import com.hedera.hapi.node.network.NetworkGetExecutionTimeQuery;
+import com.hedera.hapi.node.network.NetworkGetExecutionTimeResponse;
+import com.hedera.hapi.node.transaction.Query;
+import com.hedera.hapi.node.transaction.Response;
+import com.hedera.node.app.service.networkadmin.impl.handlers.NetworkGetExecutionTimeHandler;
+import com.hedera.node.app.spi.workflows.PreCheckException;
+import com.hedera.node.app.spi.workflows.QueryContext;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class NetworkGetExecutionTimeHandlerTest {
+    @Mock
+    private QueryContext context;
+
+    private NetworkGetExecutionTimeHandler subject;
+
+    @BeforeEach
+    void setUp() {
+        subject = new NetworkGetExecutionTimeHandler();
+    }
+
+    @Test
+    void extractsHeader() {
+        final var data = NetworkGetExecutionTimeQuery.newBuilder()
+                .header(QueryHeader.newBuilder().build())
+                .build();
+        final var query = Query.newBuilder().networkGetExecutionTime(data).build();
+        final var header = subject.extractHeader(query);
+        final var op = query.networkGetExecutionTimeOrThrow();
+        assertEquals(op.header(), header);
+    }
+
+    @Test
+    void createsEmptyResponse() {
+        final var responseHeader = ResponseHeader.newBuilder().build();
+        final var response = subject.createEmptyResponse(responseHeader);
+        final var expectedResponse = Response.newBuilder()
+                .networkGetExecutionTime(
+                        NetworkGetExecutionTimeResponse.newBuilder().header(responseHeader))
+                .build();
+        assertEquals(expectedResponse, response);
+    }
+
+    @Test
+    void validateThrowsPreCheck() {
+        assertThrows(PreCheckException.class, () -> subject.validate(context));
```

Comment:
Same here, would be good to check the response code

Reply:
Fixed